tests/mbed/host/CMakeLists.txt

```diff
@@ -7,7 +7,9 @@ oeedl_file(../mbed.edl host gen)
 add_executable(libmbedtest_host host.c ocalls.c ${gen})
 
 target_compile_definitions(libmbedtest_host PRIVATE PROJECT_DIR="${CMAKE_SOURCE_DIR}/")
-target_compile_options(libmbedtest_host PRIVATE -Wno-error)
+if (NOT WIN32)
+    target_compile_options(libmbedtest_host PRIVATE -Wno-error)
```

Comment:
Prefer 2-space indent as the default please. Also, why do we need to remove this from Windows?

Reply:
-Wno-error does not work on Windows.

Reply:
Does it need to be replaced with a Windows equivalent, or do not compile with -Werror semantics on Windows?

Reply:
No, we just remove it. This is only host test code on Windows.

Reply:
Changed this to 2-space indent.

Reply:
Not an issue.